### PR TITLE
Merge develop from main repo

### PR DESCRIPTION
**Pull request by Levia**
This branch defines the merging of the `online_reporting_tool` repo, `develop` branch

### CODE REVIEW

File: .shippable.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: ruby
```
Comment (agnessa) [inline .shippable.yml]: we no longer need shippable - we use travis in a public repo

File: .yardoc/checksums [hunk 2]
```diff
@@ -0,0 +1,153 @@
+app/models/delegation.rb aac7b3354c02759b4eecb76f272559e2a5c1bc6e
```
Comment (agnessa) [inline .yardoc/checksums]: let's remove this

Comment (Levia) [inline .yardoc/checksums]: Including `rdoc`(gem and generated files)? I'm not sure if it's the same thing we have in the ORS documentation site

Comment (agnessa) [inline .yardoc/checksums]: Yes, all the rdoc stuff as well - that was generated 3 years ago. Also the directory old_cap can go now :)

Comment (Levia) [inline .yardoc/checksums]: Cool. All done!

**Comment by agnessa** [comment]
great, thanks, glad it's done! :smiling_imp:

**Pull request merged**